server/routes/sentry/tasks.py

```
@@ -0,0 +1,420 @@
+
+                            cursor.execute(
+                                "UPDATE incidents SET rca_celery_task_id = %s WHERE id = %s",
+                                (task.id, str(incident_id)),
+                            )
+                            conn.commit()
+
+                            logger.info(
+                                "[SENTRY][WEBHOOK] Triggered background RCA for session %s (task_id=%s)",
+                                session_id, task.id,
+                            )
+                    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)